### DISCUSSION TITLE
kpanda/user-guide/clusters/runtime

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2335] [Announcements]
# kpanda/user-guide/clusters/runtime

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/clusters/runtime.html

**Comment by liyuerich** [comment]
不同操作系统和支持的运行时对应关系:
支持的Containerd 运行时版本已经更新，这里能维护最新的吗？
